### PR TITLE
Support for CosmosDB Mongo tree nodes in new connection dialog azure tree

### PR DESCRIPTION
**Pull request by languy**
This PR adds support for display CosmosDB mongo-api tree nodes in the "New Connection" dialog under "Browse" and Azure tree.
The code is disabled for now in `extensions/azurecore/src/extension.ts` until the extension is ready.

### CODE REVIEW

File: extensions/azurecore/src/azureResource/providers/cosmosdb/mongo/cosmosDbMongoTreeDataProvider.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+				id: generateGuid(),
+				connectionName: databaseServer.name,
+				serverName: databaseServer.name,
+				userName: databaseServer.loginName,
+				password: '',
+				authenticationType: 'AzureMFA',
+				savePassword: true,
+				groupFullName: '',
+				groupId: '',
+				providerName: 'COSMOSDB_MONGO',
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/azureResource/providers/cosmosdb/mongo/cosmosDbMongoTreeDataProvider.ts:48]: Where is this provider from?

File: extensions/azurecore/src/extension.ts [hunk 2]
```diff
@@ -135,6 +137,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 				new AzureResourceDatabaseProvider(new AzureResourceDatabaseService(), extensionContext),
 				new SqlInstanceProvider(new SqlInstanceResourceService(), extensionContext),
 				new PostgresServerProvider(new PostgresServerService(), extensionContext),
+				// new CosmosDbMongoProvider(new CosmosDbMongoService(), extensionContext)
```
Comment (Charles-Gagnon) [inline extensions/azurecore/src/extension.ts:140]: Just delete these until you actually enable it.

Although if this code isn't enabled then I'm curious what the point of getting this in now is. Could you not just wait until the feature is ready to do the PR?

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]
Looks generally fine but I have a few questions.

And please include screenshots of the new folders/nodes in the PR description.

**Comment by alanrenmsft** [comment]
@languy I'd recommend submitting this PR when the CosmosDB extension is ready.

**Pull request closed without merging**